File: Java/MathematicalPrograms/2DShapes/AreaAndPerimeterOfCircle.java
import java.util.Scanner;
import static java.lang.Math.*;

public class AreaAndPerimeterOfCircle {
	public static void main(String[] args) {
		Scanner myObj = new Scanner(System.in);
		System.out.println("Enter the Radius");
		int radiusper = myObj.nextInt();
		double a = PI*radiusper*radiusper;
		double perimeter = 2*PI*radiusper;
		System.out.println("Area = " + a);
		System.out.println("Perimeter = " + perimeter);
	}
}

File: C++/MathematicalPrograms/2DShapes/AreaAndPerimeterOfCircle.cpp
#include <iostream>
#include <math.h>
#include <string>

using namespace std;

int main() {
	double PI = atan(1)*4;
	double radius;
	cout << "Enter the Radius";
	cin >> radius;
	double area = PI*radius*radius;
	double perimeter = 2*PI*radius;
	cout << "Area = " << area;
	cout << "\nPerimeter = " << perimeter;
}